declarations/index.d.ts

```diff
@@ -0,0 +1,2 @@
+export = webpack;
+declare const webpack: (options: import("../declarations/WebpackOptions.js").WebpackOptions | import("../declarations/WebpackOptions.js").WebpackOptions[], callback?: import("./webpack.js").Callback<import("./Stats.js") | import("./MultiStats.js")>) => import("./Compiler.js") | import("./MultiCompiler.js");
```

Comment:
Many attached types are missing here.

This seem to be a typescript bug.

Source file: 

https://github.com/webpack/webpack/blob/df53aeaa0e10c7c342709344712150985699b1a8/lib/index.js#L19-L27

Reply:
The same pattern seem to work when classes are exported. Seem to be only an issue when a function is exported via `module.exports`

Reply:
But it works for a function here: declarations/util/compileBooleanMatcher.d.ts